### PR TITLE
[NUI][Xaml] Fix issue that BindingContext can't be used in C# code （JIRA 2296）

### PR DESCRIPTION
**Pull request by AdunFang**
### API Changes ###

 - ACR:

### CODE REVIEW

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by wb0716 — APPROVED** [review]
Good for me

File: src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs [hunk 1]
```diff
@@ -36,14 +36,39 @@ public abstract class BindableObject : INotifyPropertyChanged, IDynamicResourceH
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static readonly BindableProperty BindingContextProperty =
-            BindableProperty.Create(nameof(BindingContext), typeof(object), typeof(BindableObject), default(object),
-                                    BindingMode.OneWay, null, BindingContextPropertyChanged, null, null, BindingContextPropertyBindingChanging);
+            BindableProperty.Create(nameof(BindingContext), typeof(object), typeof(BindableObject), null, propertyChanged: (BindableProperty.BindingPropertyChangedDelegate)((bindable, oldValue, newValue) =>
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs:39]: could you check this?

https://github.com/Samsung/TizenFX/pull/3122#discussion_r642180063

thank you.

**Review by elishateng — APPROVED** [review]
LGTM

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**